FHEM/00_SIGNALduino.pm

```diff
@@ -2115,6 +2126,9 @@ SIGNALduino_Parse_MS($$$$%)
 				#Debug $patternLookupHash{substr($rawData,$i,$signal_width)}; ## Get $signal_width number of chars from raw data string
 				if (exists $patternLookupHash{$sig_str}) { ## Add the bits to our bit array
 					push(@bit_msg,$patternLookupHash{$sig_str})
+				} elsif ((length($sig_str)< $signal_width ) && (exists $endPatternLookupHash{$sig_str})) {
+					SIGNALduino_Log3 $name, 5, "$name: letztes Bit rekonstruiert.";
```

Comment:
Logmeldungen  sind üblicherweise immer in englisch.

`SIGNALduino_Log3 $name, 5, "$name: last bit reconstructed";
`